internal/repositories/aftermarket_devices_test.go

```diff
@@ -134,3 +134,54 @@ func Test_GetOwnedAftermarketDevices_Pagination_PreviousPage(t *testing.T) {
 	}
 	assert.Exactly(t, expected, res.Edges)
 }
+
+func Test_GetAftermarketDevices_FilterByBeneficiary(t *testing.T) {
```

Comment:
Some missing tests

- When beneficiary has no records (**another beneficary's id is supplied that has no records**)
- Both Owner and Beneficiary are supplied

Reply:
I'm okay to go without these given how orthogonal the code is with everything else. Let's focus on tests for the other PR, which is a bit more intricate.